nuxt.schema.ts

```diff
@@ -55,6 +55,22 @@ export default defineNuxtSchema({
               default: 'i-heroicons-hashtag-20-solid'
             })
           }
+        }),
+        primary: field({
+          type: 'string',
+          title: 'Primary',
+          description: 'Primary color of your UI.',
+          icon: 'i-mdi-palette-outline',
+          default: 'sky',
```

Comment:
Why not set the defaults like in `app.config.ts`? https://github.com/nuxt-ui-pro/docs/blob/main/app.config.ts#L3-L4